test/model/content_test.dart

```diff
@@ -521,6 +521,21 @@ class ContentExample {
       blockUnimplemented('more text'),
     ]]),
   ]);
+
+  static const lineBreaksAfterCodeBlocks = ContentExample(
+    'blank text nodes after code blocks',
+    '    code block''\n\n''some content',
+    '<div class="codehilite">'
+      '<pre><code>'
```

Comment:
Hmm, when I do it, I get something different here:

```suggestion
      '<pre><span></span><code>'
```

Here's my message: https://chat.zulip.org/#narrow/stream/7-test-here/topic/content/near/1770526